src/htmlContent/main-view.html

```diff
@@ -1,22 +1,22 @@
-<!-- 
```

Comment:
All these changes are unrelated to your patch.  It looks like your editor is set to automatically remove end-of-line-whitespace.  You should disable that, and only do it manually (or get one that only does it for lines you edit).  Changing non-related lines is bad practice in open source, because it masks git blame information about when the code last changed, and who changed it.